### PR TITLE
Fix broken docs

### PR DESCRIPTION
**Pull request by aorenste**
These were causing doctest to be unhappy.

In particular the doc from #124496 caused #124771 to fail "trunk / win-vs2019-cpu-py3 / test" to fail when pushing. Not sure why it wasn't a problem on the original PR.

Testing:

`./test/run_doctests.sh`:
  before:
```
=== 4 warnings in 11.21 seconds ===
```
  after:
```
===  in 11.11 seconds ===
```

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #124940

### CODE REVIEW

**Review by zou3519 — APPROVED** [review]
interesting, thank you

File: torch/library.py [hunk 1]
```diff
@@ -853,35 +853,38 @@ def opcheck(
 
     Example:
 
-        >>> # xdoctest: +REQUIRES(env:TORCH_DOCTEST_CUDA)
-        >>> @torch.library.custom_op("mylib::numpy_mul", mutates_args=())
-        >>> def numpy_add(x: Tensor, y: float) -> Tensor:
-        >>>     x_np = x.numpy(force=True)
-        >>>     z_np = x_np + y
-        >>>     return torch.from_numpy(z_np).to(x.device)
+        >>> def example():
```
Comment (zou3519) [inline torch/library.py:856]: Why do we need to put this into an example()?

Comment (aorenste) [reply torch/library.py:856]: With xdoctest v1.0.2 (which is what is installed on win-vs2019-cpu-py3) having the REQUIRES wrap a decorator seems to not work (it gets confused and complains about "key not found"). This seems to be fixed in later version of xdoctest - but wrapping in an example is a work-around.

Comment (zou3519) [reply torch/library.py:856]: Can we just disable / ignore xdoctest on windows temporarily, or upgrade it? We don't want users to write code with this wrapping.

Comment (aorenste) [reply torch/library.py:856]: Let me see if I can figure out how to force a newer version...

**Review by atalman — APPROVED** [review]

**Review by huydhn — APPROVED** [review]
LGTM!  Just FYI, the list of Windows pip dependencies is here https://github.com/pytorch/test-infra/blob/main/aws/ami/windows/scripts/Installers/Install-Pip-Dependencies.ps1#L16, but we haven't updated it in a while because of the time taken to roll out a new Windows AMI.  

@atalman Do you think we should it's consolidate some few left over dependencies here before you roll out the new Windows AMI?  They should be safe to include because they are only CI dependencies

**Comment by huydhn** [comment]
> @atalman Do you think we should it's consolidate some few left over dependencies here before you roll out the new Windows AMI? They should be safe to include because they are only CI dependencies

Oops, I'm too late, the new AMI is already there I https://github.com/pytorch-labs/pytorch-gha-infra/pull/382.  It's for the next time then

**Pull request closed without merging**